src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,193 @@
+import cn from 'classnames';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  Link,
+  useLocation,
+  useParams,
+  useSearchParams,
+} from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { columns } from './colums';
+
+type Props = {
+  loadedPeople: Person[]
+};
+
+export const PeopleTable:React.FC<Props> = ({ loadedPeople }) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [visiblePersons, setVisiblePersons] = useState<Person[]>([]);
+  const { personSlug } = useParams();
+  const { search } = useLocation();
+
+  const sortOrder = searchParams.get('order');
+  const sortType = searchParams.get('sort');
+
+  const getSortType = useCallback((name: string): string => {
+    switch (true) {
+      case name === sortType && sortOrder === 'asc':
+        return 'fas fa-sort-up';
+      case name === sortType && sortOrder === 'desc':
+        return 'fas fa-sort-down';
+      default:
+        return 'fas fa-sort';
+    }
+  }, [sortType, sortOrder]);
```

Comment:
let's move it to a separate helper file. `Switch/cases` don't look too good in the main logic